### ISSUE TITLE
Wrong ns plugin version can be loaded

### ISSUE THREAD

**Issue by XinyuHou** [synergy/synergy#4866]
1. Install old version
2. Download ns plugin via wizard
3. Update to new version
4. Skip wizard

Actual: Old plugin is loaded without warning
Expect: Warning and plugin shouldn't be loaded

**Comment by XinyuHou** [comment]
Fixed by 41013091baacd13859caf22ae6bc95b53744f455

**Comment by nbolton** [comment]
This logging isn't very helpful:
* It doesn't tell you what version it has or expects.
* The "skip plugin" message is ambiguous (and kind of useless).
* The "crypto disabled" message should be a warning.

```
[2015-07-23T16:41:08] ERROR: plugin version doesn't match
[2015-07-23T16:41:08] ERROR: skip plugin: ns.dll
[2015-07-23T16:41:08] NOTE: crypto disabled because of ns plugin not available
```
